compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
+
+#### Short syntax
+
+The short syntax variant only specifies the secret name. This grants the
+container access to the secret and mounts it at `/run/secrets/<secret_name>`
+within the container. The source name and destination mountpoint are both set
+to the secret name.
+
+> **Warning**: Due to a bug in Docker 1.13.1, using the short syntax currently
+> mounts the secret with permissions `000`, which means the secret is unreadable
+> if you define it using the short syntax. The workaround is to use the long
```

Comment:
It's only uneradable if you use a non-root user. Since the root user is default, many people will never see this bug. Can we update to say "unreadable by non-root users"?